src/storage/src/source/postgres/replication.rs

```diff
@@ -748,15 +770,14 @@ fn extract_transaction<'a>(
     ssh_tunnel_manager: &'a SshTunnelManager,
     metrics: &'a PgSourceMetrics,
     publication: &'a str,
-    errored_outputs: &'a mut HashSet<usize>,
-) -> impl AsyncStream<
-    Item = Result<(u32, usize, Result<Vec<Option<Bytes>>, DefiniteError>, Diff), TransientError>,
-> + 'a {
+    errored_tables: &'a mut HashSet<u32>,
```

Comment:
To match the previous semantics once a table is dropped or truncated we stop emitting any events for it, so we likely need to update the first few match statements to ignore events for errored tables:
```
Insert(body) if !table_info.contains_key(&body.rel_id()) || errored_tables.contains(&body.rel_id) => continue,
...
```